places.go

```diff
@@ -506,6 +506,14 @@ func (r *PlaceDetailsRequest) params() url.Values {
 		q.Set("fields", strings.Join(placeDetailsFieldMasksAsStringArray(r.Fields), ","))
 	}
 
+	if uuid.UUID(r.SessionToken).String() != "00000000-0000-0000-0000-000000000000" {
```

Comment:
seems awkward

there's no `.IsZero()` method on UUID? :|

Reply:
Nope, no `.IsZero()`. I can't see anything that would be more idiomatic in the UUID documentation.